README.md

```diff
@@ -16,10 +16,10 @@ Ubuntu 16.04 | ami-42b86d54 | ami-7fa85269
 Ubuntu 16.10 | ami-0c86521a | ami-4a84505c
 
 ## Pre-requisites
-
+This first command works in bash but not ZSH
```

Comment:
we can delete this comment. In my other PR I address the change I had to make to add quotes to make the rake tasks work